test/extended/util/framework.go

```diff
@@ -275,6 +286,17 @@ func ExtendedTestPath() string {
 	return os.Getenv("EXTENDED_TEST_PATH")
 }
 
+//ArtifactDirPath returns the value of ARTIFACT_DIR environment variable
+func ArtifactDirPath() string {
+	return os.Getenv("ARTIFACT_DIR")
+}
```

Comment:
ok i'll retract that one, i see you're following a pattern that's been established (a dubious pattern, but..)